app/views/type-2-diabetes/information-and-support-type-2-diabetes.html

```diff
@@ -68,7 +68,7 @@
 ##Keeping active with type 2 diabetes
 
 - [Getting active and staying active](https://www.diabetes.org.uk/Guide-to-diabetes/Managing-your-diabetes/Exercise/) - the charity Diabetes UK explains what kind of activity you could do and gives tips on how to get active
-- [Video with exercise tips](https://www.youtube.com/watch?v=9RDltBjdbEY) - simple exercises that you can do at home
+- [Video with exercise tips](http://www.nhs.uk/video/Pages/keep-active-150-minutes-exercise.aspx?searchtype=Tag&searchterm=Healthy+living__Fitness&offset=17&) - simple exercises that you can do at home
```

Comment:
Can you drop all the query parameters from the URL? This seems to link to the right place still: http://www.nhs.uk/video/Pages/keep-active-150-minutes-exercise.aspx